pkg/network/http/model.go

```diff
@@ -40,6 +40,17 @@ func (k *httpBatchKey) Prepare(n httpNotification) {
 	k.page_num = C.uint(int(n.batch_idx) % HTTPBatchPages)
 }
 
+func (tx *httpTX) postProcess() {
+	b := *(*[HTTPBufferSize]byte)(unsafe.Pointer(&tx.request_fragment))
+	for i := tx.fragmentSize(); i < HTTPBufferSize; i++ {
+		b[i] = 0
+	}
+}
+
+func (tx *httpTX) fragmentSize() int {
+	return int(tx.fragment_len)
```

Comment:
could we add telemetry like (h.telemetry.malformed.Inc() / statKeeper:processHTTPPath()) when we see unrealistic fragment size, may later, probably in httpStatKeeper

Reply:
What is an unrealistic value?